src/gui/processing/models/qgsmodelarrowitem.cpp

```diff
@@ -81,30 +82,31 @@ void QgsModelArrowItem::paint( QPainter *painter, const QStyleOptionGraphicsItem
   else
     color.setAlpha( 80 );
 
+  //
   QPen p = pen();
   p.setColor( color );
-  p.setWidth( 1 );
+  p.setWidth( 0 );
   painter->setPen( p );
+
   painter->setBrush( color );
   painter->setRenderHint( QPainter::Antialiasing );
 
-
   switch ( mStartMarker )
   {
-    case Marker::Circle:
-      painter->drawEllipse( mStartPoint, 3.0, 3.0 );
-      break;
     case Marker::ArrowHead:
       drawArrowHead( painter, mStartPoint, path().pointAtPercent( 0.0 ) - path().pointAtPercent( 0.05 ) );
       break;
+
+    // start marker are no longer drawn
+    case Marker::Circle:
     case Marker::NoMarker:
       break;
   }
 
   switch ( mEndMarker )
   {
     case Marker::Circle:
-      painter->drawEllipse( mEndPoint, 3.0, 3.0 );
+      painter->drawEllipse( mEndPoint, 3, 3 );
```

Comment:
There's no benefit to this change -- the QPainter method takes a qreal argument, so it'll be converted from int anyway.